### ISSUE TITLE
Language cannot be changed using Voice Control

### ISSUE THREAD

**Issue by ValentinaPC** [mozilla-mobile/mozilla-vpn-client#670]
#### Affected versions:
* v 2.0.4 (2.202103101447)

      1. Affected Platforms:

* macOS 10.15.7;

      1. Prerequisites:

* Have the VPN app installed;
* Have a FxA account with a subscription made for VPN;
* Have Voice Control enabled;

      1. Steps to reproduce:

1. Open VPN app;
1. Go to Settings -> Language option;
1. Change language using Voice Control (e.g. from German to American English);

      1. Expected result:

* Language can be changed using Voice Control.

      1. Actual result:

* Language does not change using Voice Control.
![Voice-control-2](https://user-images.githubusercontent.com/15685960/110790840-3020a980-827a-11eb-9938-54f7e0df65c2.gif)

┆Issue is synchronized with this [Jira Bug](https://mozilla-hub.atlassian.net/browse/VPN-118)
┆Reporter: Data Sync

**Comment by data-sync-user** [comment]
➤ Valentina Virlics commented:

Verified this as fixed while using Mozlla VPN 2.15.0 (2.202305231658) build on MacOS 13;

User is able to change app language using Voice control.
